### PR TITLE
return boolean for app online helper

### PR DESCRIPTION
**Pull request by jarosser06**

### CODE REVIEW

File: libraries/helpers.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def resource_attrs
 
 def pm2_is_app_online?(name)
   cmd = shell_out!('pm2 list', :returns => 0)
-  cmd.stdout =~ /#{name}/
+  not cmd.stdout.match(name).nil?
```
Comment (miguelcnf) [inline libraries/helpers.rb]: Would you mind just changing 'not' with '!' so that Rubocop is happy? Thanks for this by the way.

**Comment by jarosser06** [comment]
Fixed. :)

**Comment by miguelcnf** [comment]
Merged, will release a new version today. Thanks again.

**Comment by miguelcnf** [comment]
Released 0.2.1.

**Pull request merged**